From c2416e1996f694ea0cefb217f806e45931ef0b03 Mon Sep 17 00:00:00 2001
From: Dylan Reid <dgreid@chromium.org>
Date: Wed, 15 Feb 2017 12:31:55 -0800
Subject: [PATCH] Make city.h includes consistent

Most of the code base already uses <misc/city.h>.  Modify the places
that include "city.h" to follow that pattern.  cmake doesn't add misc to
the include paths passed to the compiler.  I ran into "can't find
city.h" errors when attempting to add an ebuild for gentoo.

Change-Id: I1e07d82bbe58fbae4759263734ac22170220eeb7
Signed-off-by: Dylan Reid <dgreid@chromium.org>
---
 ntirpc/misc/citycrc.h | 2 +-
 src/city-test.c       | 2 +-
 src/citycrc.h         | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/ntirpc/misc/citycrc.h b/ntirpc/misc/citycrc.h
index 57069344f1f8..d3620158a5ca 100644
--- a/ntirpc/misc/citycrc.h
+++ b/ntirpc/misc/citycrc.h
@@ -34,7 +34,7 @@
 #ifndef CITY_HASH_CRC_H_
 #define CITY_HASH_CRC_H_
 
-#include "city.h"
+#include <misc/city.h>
 
 /* Hash function for a byte array. */
 uint128 CityHashCrc128(const char *s, size_t len);
diff --git a/src/city-test.c b/src/city-test.c
index 03d98a2ba9ec..8b504dd5c57c 100644
--- a/src/city-test.c
+++ b/src/city-test.c
@@ -27,7 +27,7 @@
 #include <string.h>
 #include <stdio.h>
 
-#include "city.h"
+#include <misc/city.h>
 #ifdef __SSE4_2__
 #include "citycrc.h"
 #endif
diff --git a/src/citycrc.h b/src/citycrc.h
index f6cab94eff04..7998521b68cf 100644
--- a/src/citycrc.h
+++ b/src/citycrc.h
@@ -33,7 +33,7 @@
 #ifndef CITY_HASH_CRC_H_
 #define CITY_HASH_CRC_H_
 
-#include "city.h"
+#include <misc/city.h>
 
 /* Hash function for a byte array. */
 uint128 CityHashCrc128(const char *s, size_t len);
-- 
2.11.1

